.devcontainer/post-create.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This script runs after the dev container is created
+# It performs additional dependency pre-loading to speed up builds
+
+echo "Setting up OpenTelemetry Java Instrumentation development environment..."
+
+# Ensure we're in the right directory
+cd /workspace
+
+# Set up environment variables for faster builds
+export GRADLE_OPTS="-Dorg.gradle.daemon=true -Dorg.gradle.parallel=true -Dorg.gradle.caching=true"
+
+# Download Gradle wrapper and start daemon
+echo "Starting Gradle daemon..."
+./gradlew --version
+
+# Try to run the custom resolveDependencies task with timeout
+echo "Pre-downloading some dependencies..."
+timeout 180 ./gradlew resolveDependencies --no-daemon --parallel || echo "Dependency resolution completed or timed out"
```

Comment:
what about using ./gradlew assemble, which will also populate the gradle build cache

and why timeout needed?